test/test-util/src/kafka/kafka_client.rs

```diff
@@ -26,10 +26,20 @@ pub struct KafkaClient {
 }
 
 impl KafkaClient {
-    pub fn new(kafka_url: &str, group_id: &str, topic: &str) -> Result<KafkaClient, anyhow::Error> {
+    pub fn new(
+        kafka_url: &str,
+        group_id: &str,
+        topic: &str,
+        configs: Option<&[(&str, &str)]>,
```

Comment:
I don't think it needs to be an `Option` -- you could just have it be `&[(&str, &str)]` and users that don't want to set any configs could pass an empty slice.